telemetry.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import errors
+import requests
+
+
+__version__ = '0.0.1'
+URI = os.environ['TELEMETRY_URI']
+TOKEN = os.environ['TELEMETRY_TOKEN']
+
+
+_ADDITIONAL_REQUEST_ARGUMENTS = {}
+if 'TELEMETRY_CA_CERT' in os.environ:
```

Comment:
so if this isn't provided, does anything bad happen?

Reply:
> so if this isn't provided, does anything bad happen?

We just die on `CERTIFICATE_VERIFY_FAILED`.  But if you point your bot at a `TELEMETRY_URI` that does not require a custom CA, you don't need to set `TELEMETRY_CA_CERT`.

Reply:
makes sense. ty!